Windows support

Windows doesn't seem to like the slashes.  When using resolvify, I get things like:
```
Error: Cannot find module 'C:ProjectsPersonalPatterns
ode_moduleabel-polyfilllibindex.js' from 'C:\Projects\Personal\Patterns\src'
```
(Notice the '\' missing and the 'n' is missing from 'node_modules'... that is because `\n` is a end-of-line... so that's what it did; similar with `\b`)
When the first path is supposed to be `C:\Projects\Personal\Patterns\node_module\babel-polyfill\lib\index.js`
Works fine on Mac.  Also works on windows without this transform, (manually changing paths to be relative) which leads me to believe this transform to be the issue.

When creating the path string, you probably want to escape the forward slashes

--- a/index.js
+++ b/index.js
@@ -11,7 +11,7 @@ var transform = transformTools.makeRequireTransform(
     function(args, opts, cb) {
         var paths = Object.keys(opts.config)
                         .filter(function(opts_key) { return opts_key.length == 1 })
-                        .reduce(function(coll, opts_key) { 
+                        .reduce(function(coll, opts_key) {
                             return coll.concat(opts.config[opts_key])
                         },[])
         if (paths.length == 0) { return cb(null) }
@@ -20,7 +20,7 @@ var transform = transformTools.makeRequireTransform(
             moduleDirectory: ['node_modules'].concat(paths)
         });
 
-        return cb(null, "require('"+resolved+"')");
+        return cb(null, "require('"+resolved.replace(/\\/g,'\\\\')+"')");
     }
 );
 

